UT4MasterServer/Authentication/BasicAuthenticationHandler.cs

```diff
@@ -26,17 +31,25 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
 				return AuthenticateResult.Fail("unexpected scheme");
 			}
 
-			var client = new ClientIdentification(authorization.Value);
-			if (client.ID.IsEmpty || client.Secret.IsEmpty)
+			var clientParser = new ClientIdentification(authorization.Value);
+			if (clientParser.ID.IsEmpty || string.IsNullOrEmpty(clientParser.Secret))
```

Comment:
IMO better is `IsNullOrWhitespace`

Reply:
I was also considering this, but i ultimately decided on `IsNullOrEmpty` because if we create a Client with just whitespace (very unlikely) as secret we still want to handle it properly.